Prototyper/Classes/Feedback/FeedbackView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FeedbackView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 20.12.19.
+//
+
+import SwiftUI
+
+struct FeedbackView: View {
+    @EnvironmentObject var model: Model
+    @State var descriptionText: String = ""
+    @State var showSheet: Bool = false
+    @State var showScreenshot: Bool = true
+    @State var activeSheet: ActiveSheet = .loginSheet
+    @State var showSendFeedbackView: Bool = false
+    @State var feedback: Feedback?
+    
+    enum ActiveSheet {
+        case loginSheet, MarkupSheet
+    }
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                HStack (alignment: .top) {
+                    if showScreenshot {
+                        ZStack (alignment: .topTrailing) {
+                            ZStack {
+                                Image(uiImage: model.screenshot)
+                                    .resizable()
+                                    .border(Color(white: 0.75))
+                                    .scaledToFit()
+                                Image(systemName: "hand.draw.fill")
+                                    .imageScale(.large)
+                                    .onTapGesture {
+                                        self.editImage()
+                                }
+                            }
+                            Image(systemName: "xmark.circle.fill")
```

Comment:
It seems like the X is not right on the corner of the image, can you change its alignment to be right on the corner? Maybe if we add a shadow to the image the size of the view containing the image increases by itself and this problem is solved?

Reply:
I added some offset too, which fixed the problem